<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_top_left">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_top_mid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_top_right">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mid_left">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mid_mid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mid_right">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_bot_left">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_bot_mid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_bot_right">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_restart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Who will win?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_bot_player">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Play with a computer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
